core/src/main/scala/cats/data/WriterT.scala

```diff
@@ -63,6 +63,17 @@ final case class WriterT[F[_], L, V](run: F[(L, V)]) {
     mapWritten(_ => monoidL.empty)
 
   def show(implicit F: Show[F[(L, V)]]): String = F.show(run)
+
+  def foldLeft[C](c: C)(f: (C, V) => C)(implicit F: Foldable[F]): C =
+    F.foldLeft(run, c)((z, v) => f(z, v._2))
```

Comment:
Can we call `v` instead `lv` here to remind us it has the written value?

Reply:
Sure thing, thanks